### PR TITLE
STAC-2599 Update debian repo in readme and improve gitlabci

### PR DESCRIPTION
**Pull request by TizianoPerrucci**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -3,8 +3,7 @@ image: stackstate/stackstate-agent-runner-circle:latest
 stages:
 - prepare
```
Comment (craffit) [inline .gitlab-ci.yml]: Theres a couple of environment variables that are not here:

https://gitlab.com/stackvista/stackstate-agent/settings/ci_cd

but should go in this file like here:

https://docs.gitlab.com/ee/ci/yaml/README.html#variables

PROCESS_AGENT_VERSION: Should be in the release.lock
To gitlab file:
STS_AWS_RELEASE_BUCKET, STS_AWS_TEST_BUCKET, STS_DOCKER_RELEASE_REPO, STS_DOCKER_TEST_REPO

File: .gitlab-scripts/publish_image.sh [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+cp $ARTIFACT_PATH/*.deb Dockerfiles/agent
+
+docker build -t stackstate/${IMAGE_REPO}:${IMAGE_TAG} Dockerfiles/agent
+docker tag stackstate/${IMAGE_REPO}:${IMAGE_TAG} stackstate/${IMAGE_REPO}:latest
 docker login -u $DOCKER_USER -p $DOCKER_PASS
-docker push stackstate/stackstate-agent:$TAG
-docker push stackstate/stackstate-agent:latest
+echo docker push stackstate/${IMAGE_REPO}:${IMAGE_TAG}
+docker push stackstate/${IMAGE_REPO}:${IMAGE_TAG}
+echo docker push stackstate/${IMAGE_REPO}:latest
```
Comment (craffit) [inline .gitlab-scripts/publish_image.sh]: its a bit misleading that for dev builds, the latest is the latest that was build (from any branch). I would expect this to be the latest master build

Comment (Voronenko) [reply .gitlab-scripts/publish_image.sh]: agree

File: .gitlab-ci.yml (hunk 1 above)
Comment (TizianoPerrucci) [reply .gitlab-ci.yml]: @craffit release.lock , what you mean ?

Comment (craffit) [reply .gitlab-ci.yml]: @TizianoPerrucci I meant this file: https://github.com/StackVista/stackstate-agent/blob/master/release.json. That should determine the versions we build against

Comment (TizianoPerrucci) [reply .gitlab-ci.yml]: interesting, I even didn't notice that file :sweat_smile: 
I kinda like your suggestion, but don't think this PR is the place where we should start using the release.json
lets get another ticket :) ?

Comment (Voronenko) [reply .gitlab-ci.yml]: I think, it get's injected already

Comment (craffit) [reply .gitlab-ci.yml]: Okidok, we do this some other time

**Pull request merged**